notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1391 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    IMPORT_FILE = "gs://automl-video-demo-data/hmdb_split1_5classes_train_inf.csv"
</pre><p>Needs to be in the right bucket. See notebook workflow doc for details.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>